src/test/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+#Bridge related settings
+bridge.id=my-bridge
+
+#MQTT server common
+mqtt.server.host=localhost
+mqtt.server.port=1883
+
+#Apache Kafka common
+kafka.bootstrap.servers=localhost:9092
+
+#Apache Kafka producer
+kafka.producer.acks=1
```

Comment:
Same comment as before, about removing this parameter from the configuration.

Reply:
@antonio-pedro99 the acks configuration is still here in the test, we should remove from here as well for now.

Reply:
Yes, I forgot to remove. 
I have updated it. Please take a look.